.github/workflows/python-lint.yaml

```diff
@@ -0,0 +1,13 @@
+name: python-lint
+on:
+  pull_request:
+    branches:
+      - "*"
+jobs:
+  build:
+    name: python Lint
+    runs-on: ubuntu-latest
+    steps:
```

Comment:
I think there are some missing steps here. Checkout and UV installation for example.
Maybe we can use [this](https://docs.astral.sh/uv/guides/integration/github/) action

Reply:
done